Incorrect Go pkg import comment causing downstream processing error (kythe)

### What happened?

https://github.com/helm/helm/blob/5caf0044d4ef3d62a955440272999e139aafbbed/pkg/kube/statuswait.go#L17 is in a v4 package but it says `// import "helm.sh/helm/v3/pkg/kube"`

https://github.com/helm/helm/blob/5caf0044d4ef3d62a955440272999e139aafbbed/pkg/kube/client.go#L17 has a correct one but it would be a mismatch with above.

Kythe error:

```
2026/02/19 18:31:04 ERROR: Error locating "github.com/cilium/cilium/vendor/...": louhi_ws/gopath/src/github.com/cilium/cilium/vendor/helm.sh/helm/v4/pkg/cli/environment.go:39:2: found import comments "helm.sh/helm/v4/pkg/kube" (client.go) and "helm.sh/helm/v3/pkg/kube" (statuswait.go) in /workspace/louhi_ws/gopath/src/github.com/cilium/cilium/vendor/helm.sh/helm/v4/pkg/kube
```

### What did you expect to happen?

All import comments match, or no import comment at all.

### How can we reproduce it (as minimally and precisely as possible)?

Irrelevant

### Helm version

Irrelevant

### Kubernetes version

Irrelevant

--- a/pkg/kube/statuswait.go
+++ b/pkg/kube/statuswait.go
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package kube // import "helm.sh/helm/v3/pkg/kube"
+package kube // import "helm.sh/helm/v4/pkg/kube"
 
 import (
 	"context"

